en/docs/reference/connectors/salesforce-connectors/sf-inbound-endpoint-reference-configuration.md

```diff
@@ -7,84 +7,84 @@ The following configurations allow you to configure Salesforce Inbound Endpoint
+    <td><code>37.0</code></td>
+    <td>-</td>
   </tr>
   <tr>
-    <td>userName</td>
-    <td>The username for accessing the Salesforce account.</td>
+    <td><code>connection.salesforce.userName</code></td>
+    <td>Salesforce login user name.</td>
     <td>Yes</td>
     <td>-</td>
     <td>-</td>
-  </tr> 
+  </tr>
   <tr>
-    <td>password</td>
-    <td> The password provided here is a concatenation of the user password and the security token provided by Salesforce. For more information, see <a href="https://help.salesforce.com/articleView?id=user_security_token.htm&type=5">Information on creating a security token in Salesforce</a></td>
+    <td><code>connection.salesforce.password</code></td>
+    <td>Salesforce login password.</td>
     <td>Yes</td>
-    <td>eitest123xxxxxxx</td>
+    <td><code>eitest123xxxxxxx</code></td>
     <td>-</td>
   </tr>
   <tr>
-    <td>waitTime</td>
-    <td>The time to wait to connect to the Salesforce account.</td>
+    <td><code>connection.salesforce.loginEndpoint</code></td>
+    <td>The Endpoint of the Salesforce account.</td>
     <td>Yes</td>
-    <td>5000</td>
-    <td>5 * 1000 ms</td>
-  </tr> 
+    <td><code>https://login.salesforce.com</code></td>
+    <td><code>https://login.salesforce.com</code></td>
+  </tr>
   <tr>
-    <td>connectionTimeout</td>
+    <td><code>connection.salesforce.soapApiVersion</code></td>
+    <td>The version of the Salesforce SOAP API.</td>
+    <td>No</td>
+    <td><code>22.0</code></td>
+    <td><code>22.0</code></td>
+  </tr>
+  <tr>
+    <td><code>connection.salesforce.connectionTimeout</code></td>
     <td>The time to wait to connect to the client.</td>
-    <td>Yes</td>
-    <td>20000</td>
-    <td>20 * 1000 ms</td>
-  </tr> 
+    <td>No</td>
+    <td><code>20000</code></td>
+    <td><code>20000</code></td>
+  </tr>
   <tr>
-    <td>soapApiVersion</td>
-    <td>The version of the Salesforce SOAP API.</td>
-    <td>Yes</td>
-    <td>22.0</td>
+    <td><code>connection.salesforce.waitTime</code></td>
+    <td>The time to wait to connect to the Salesforce account.</td>
+    <td>No</td>
+    <td><code>5000</code></td>
+    <td><code>5000</code></td>
+  </tr>
+  <tr>
+    <td><code>connection.salesforce.replay</code></td>
+    <td>Enabling this will read the event ID stored in the Registry or from the text file provided via the Event ID File Path parameter.</td>
+    <td>No</td>
     <td>-</td>
+    <td><code>False</code></td>
   </tr>
   <tr>
-    <td>EventIDStoredFilePath</td>
+    <td><code>connection.salesforce.EventIDStoredFilePath</code></td>
     <td>When replay is enabled, do not define any value for this property (i.e., keep it blank), to replay from the last event ID stored in the config Registry DB (property- name of the Salesforce object (follow the example below for more understanding) resource path - connector/salesforce/event). When replay is enabled, specify the directory path of a text file to start replaying from the event ID stored in it.</td>
```

Comment:
Shall we rephrase this line for more clarity? Also, let's not use the word `below`.